crates/subspace-proof-of-space/src/chiapos/table/tests.rs

```diff
@@ -18,30 +19,35 @@ fn to_chia_seed(seed: &Seed) -> Seed {
     chia_seed
 }
 
-#[cfg(target_pointer_width = "64")]
 #[test]
-fn test_compute_f1_k35() {
-    const K: u8 = 35;
+fn test_compute_f1_k25() {
```

Comment:
How do you run this test? It doesn't show up in `cargo test`, says `running 0 tests` for me :(

Reply:
```bash
cargo test --features=chia,parallel
```